plugins/circleci/api-report.md

```diff
@@ -5,81 +5,23 @@
 ```ts
 /// <reference types="react" />
 
-import { ApiRef } from '@backstage/core-plugin-api';
```

Comment:
Note the changes in this file, it's arguably the most important one of a pull request. This is the user facing changes in terms of API surface. You seem to have stripped out basically the entire interface including the API ref and route refs. Was this really the intent? It's a pretty significant change.

If you want to retain some of these, they may be worth calling out in the changeset as well so that plugin users understand the impact and what they need to do to counteract it in their own repos.

Reply:
I think I didn't fully grasp the importance of this file. I have restored the API interface and route refs.